<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Car</class>
 <widget class="QWidget" name="Car">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1202</width>
    <height>646</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QPushButton" name="MainButton">
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>130</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 170, 127);
color: rgb(255, 255, 255);
font: 25 9pt &quot;Bahnschrift Light&quot;;
font: 10pt &quot;华文隶书&quot;;
border-radius:7px;</string>
   </property>
   <property name="text">
    <string>主页</string>
   </property>
  </widget>
  <widget class="QPushButton" name="messageButton">
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>70</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 170, 127);
color: rgb(255, 255, 255);
font: 25 9pt &quot;Bahnschrift Light&quot;;
font: 10pt &quot;华文隶书&quot;;
border-radius:7px;</string>
   </property>
   <property name="text">
    <string>车辆信息</string>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stack">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1091</width>
     <height>661</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="main">
    <widget class="QLabel" name="label_title">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>420</y>
       <width>221</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>主页</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_video">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>431</width>
       <height>291</height>
      </rect>
     </property>
     <property name="text">
      <string>视频</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Car_idinput">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>270</y>
       <width>181</width>
       <height>51</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>车牌号</string>
     </property>
    </widget>
    <widget class="QPushButton" name="submit">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>360</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>入库</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="message">
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>140</y>
       <width>291</width>
       <height>241</height>
      </rect>
     </property>
     <property name="text">
      <string>车辆信息</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="ctrol">
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>30</y>
       <width>171</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 20pt &quot;华文隶书&quot;;</string>
     </property>
     <property name="text">
      <string>管理员信息</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="tablectrol">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>130</y>
       <width>651</width>
       <height>251</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
font: 25 10pt &quot;华文隶书&quot;;
border-radius:7px;</string>
     </property>
     <property name="rowCount">
      <number>6</number>
     </property>
     <property name="columnCount">
      <number>5</number>
     </property>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
    </widget>
    <widget class="QPushButton" name="ButtonAdd">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>470</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 170, 127);
color: rgb(255, 255, 255);
font: 25 9pt &quot;Bahnschrift Light&quot;;
font: 10pt &quot;华文隶书&quot;;
border-radius:7px;</string>
     </property>
     <property name="text">
      <string>添加新用户</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ButtonDelete">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>470</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 170, 127);
color: rgb(255, 255, 255);
font: 25 9pt &quot;Bahnschrift Light&quot;;
font: 10pt &quot;华文隶书&quot;;
border-radius:7px;</string>
     </property>
     <property name="text">
      <string>删除用户</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonModify">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>470</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 170, 127);
color: rgb(255, 255, 255);
font: 25 9pt &quot;Bahnschrift Light&quot;;
font: 10pt &quot;华文隶书&quot;;
border-radius:7px;</string>
     </property>
     <property name="text">
      <string>修改密码</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="CtrolButton">
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>10</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 170, 127);
color: rgb(255, 255, 255);
font: 25 9pt &quot;Bahnschrift Light&quot;;
font: 10pt &quot;华文隶书&quot;;
border-radius:7px;</string>
   </property>
   <property name="text">
    <string>管理页面</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
